MainRepository/Program.cs

```diff
@@ -1,7 +1,13 @@
-﻿internal class Program
-{
-    private static void Main(string[] args)
-    {
-        Console.WriteLine("Hello, World!");
-    }
-}
\ No newline at end of file
+﻿using System.Text;
+// Reduce string
+Console.Write ("Enter the string: ");
+string s = Console.ReadLine ();
+StringBuilder reduceString = new StringBuilder ();

```

Comment:
The 'new ()' operator can be simplified (on right side). Or you could use 'var' to implicitly declare the data type (on left side)

![image](https://github.com/karthick-1205/Assignments/assets/16486802/476606dd-a30f-4af0-8466-e48c18962638)